packages/libs/bootstrap-lib/.npmignore

```diff
@@ -0,0 +1,13 @@
+# Ignore source, test, and Storybook files.
+src/
+lib/test*/
+*.test.*
+*.stories.*
```

Comment:
@ash-vd do we have storybook in here?

Reply:
We don't, and although I don't think we will ever do in a lib, it doesn't hurt to ignore them anyway. On the other hand it would be cleaner to only ignore what's actually there... 

Now that I think of it I like the clean approach more, so only ignore files that make sense for the specific package.